### ISSUE TITLE
Feature request: Allow adding/removing items

### ISSUE THREAD

**Issue by cristoper** [LouisDotCom/NextItem#2]
It would be nice if there were a way to add and remove items.

**Comment by LouisDotCom** [comment]
Yes. I agree with both comments whole heartedly. Would like the items in a table with several columns for additional logic--priority, time of day, and things like that. I'd like to prepare some generic items only to allow the user to edit them. 



**Beamed from my Communicator.**
